cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -49,7 +49,7 @@
   if node['cdo-apps']['daemon']
     if node.chef_environment == 'staging' && node.name == 'staging' # 'real' staging only
       cronjob at:'@reboot', do:home_dir('.dropbox-dist', 'dropboxd')
-      cronjob at:'*/5 * * * *', do:deploy_dir('bin', 'cron', 'import_google_sheets')
+      cronjob at:'*/5 * * * *', do:deploy_dir('bin', 'cron', 'import_google_sheets'), notify: 'dev+build@code.org'
```

Comment:
@balderdash Does this mean the email goes out every time import_google_sheets runs or only when it fails?  I don't want to send one every 5 minutes...

Reply:
I think it will only send an email if `import_google_sheets` prints any output. If it the script exits silently when it runs successfully, then there won't be an email.

I'm not sure what the `PEGASUS_RAKE_LOGGER=true` on line 7 of `import_google_sheets` means, but if it makes those commands log to a file instead of printing to stdout, then that would be perfect.